Source/WebCore/platform/mediarecorder/MediaRecorderPrivateGStreamer.cpp

```diff
@@ -328,6 +359,25 @@ void MediaRecorderPrivateBackend::setSink(GstElement* element)
     m_sink = element;
 }
 
+void MediaRecorderPrivateBackend::configureAudioEncoder(GstElement* element)
+{
+    if (!gstObjectHasProperty(element, "bitrate")) {
+        GST_WARNING_OBJECT(m_pipeline.get(), "Audio encoder %" GST_PTR_FORMAT " has no bitrate property, skipping configuration", element);
+        return;
+    }
+
+    auto bitrate = [options = m_options]() -> int {
+        if (options.audioBitsPerSecond)
+            return *options.audioBitsPerSecond;
+        if (options.bitsPerSecond)
+            return *options.bitsPerSecond;
+        return 0;
+    }();
```

Comment:
Why convoluting this?

Reply:
This is also how the video bitrate is figured out, I don't think it's convoluted, this pattern is used elsewhere in WebKit :)

Reply:
The patter of creating a lambda with a capture that you immediately run to get a result? Might be used in places but for me it would be easier to just see a simple `if`.

Reply:
Done!